lib/services/secure_storage_service.dart

```diff
@@ -32,21 +33,21 @@ class SecureStorage {
   }
 
   saveData(String key, String val) async {
-    print("Saving secure data '$key' as '$val'");
+    logger.d("Saving secure data '$key' as '$val'");
```

Comment:
```suggestion
    logger.d("Saving secure data '$key' ");
```

We don't want the value to be printed anymore since its a security breach